modules/ROOT/pages/index.adoc

```diff
@@ -4,36 +4,21 @@ include::_attributes.adoc[]
 endif::[]
 :keywords:
 
-Design Center is a development environment for Mule applications and API definitions in the cloud. To access Design Center, you need the following things:
+Design Center is a development environment that consists of two tools:
 
+* xref:design-create-publish-api-specs.adoc[API Designer]
++
+In API Designer, you can create API specifications in RESTful API Modeling Language (RAML) 0.8 or 1.0, or according to OpenAPI specification 2.0 (OAS). You can also create API fragments in RAML. After you create a specification or fragment, you can publish it to Anypoint Exchange, so that it can be used by anyone in your MuleSoft organization.
```

Comment:
```suggestion
In API Designer, you can create API specifications in RESTful API Modeling Language (RAML) 0.8 or 1.0, or according to OpenAPI specification (OAS) 2.0. You can also create API fragments in RAML. After you create a specification or fragment, you can publish it to Anypoint Exchange, so that it can be used by anyone in your MuleSoft organization.
```
Does the organization have to be a MuleSoft organization? I ask only because I so rarely see this referred to this way....